lib.sh

```diff
@@ -709,6 +709,8 @@ open_port() {
     if ! upnpc -a "$ADDRESS" "$1" "$1" "$2" &>/dev/null
     then
         msg_box "Failed to open port $1 $2 automatically. You have to do this manually."
+        apt-get purge miniupnpc -y
+        apt autoremove -y
```

Comment:
I was planning to integrate this into check_open_port. Isn't this a bit overkill to install and uninstall this after every command? e.g. for opening port 443 and 80 we would need to call the function 2 times...

Reply:
We open the port with this function, and then check if it's open with another function.

I'm a bit tired now though, been a long day!

Reply:
I don't want any extra leftovers in the VM, that's what --^ about

Reply:
Yes, I can understand this but it will be a performance problem if we install and deinstall it every time the function gets called. (I mean it just gets installed if you call the function, which is on `no `by default)